types/gsap/Animation.d.ts

```diff
@@ -1,89 +1,103 @@
 declare namespace gsap {
-    export class Animation {
-        /** Base class for all TweenLite, TweenMax, TimelineLite, and TimelineMax classes, providing core methods/properties/() => voidality, but there is no reason to create an instance of this class directly. */
+    class Animation {
+        /* Base class for all TweenLite, TweenMax, TimelineLite, and TimelineMax classes, providing core methods/properties/() => voidality,
```

Comment:
All these comments are normal comments and earlier they were jsdoc comments. Please change them back to jsDoc comments.

Reply:
@sheetalkamat All JSDoc comments were re-added and correctly formatted.

Reply:
@sheetalkamat All JSDoc comments were re-added and correctly formatted.

Reply:
@sheetalkamat All JSDoc comments were re-added and correctly formatted.